portfolio/src/main/webapp/script.js

```diff
@@ -24,37 +25,108 @@ function getCommentLimit() {
   return selector.options[selector.selectedIndex].value;
 }
 
+function initPortfolio() {
+  fetch(authCheckURL)
+  .then(response => response.json())
+  .then(info => {
+    if (info.loggedIn) {
```

Comment:
This is a client-side check, and it's very easy to avoid by someone who wants to see the comments. If you want only logged in people to see the comments, check it on server side.

Reply:
Ah yes I noticed this and fixed it on another branch, I shall fix this here and push.